### PR TITLE
Firefox support

### LINKED ISSUE #1187: Default server works for Firefox
**Is your feature request related to a problem? Please describe.**
default server firefox won't work due to CORS settings

**Describe the solution you'd like**
use bundled canvaskit and add help documents

### PR DESCRIPTION
**Pull request by LemonHX**
## Changes

Fixes #1187

## Checklist

- [x] An issue to be fixed by this PR is listed above.
- [ ] New tests are added to ensure new features are working. End-to-end tests are usually in the `./frb_example/pure_dart` example, more specifically, `rust/src/api.rs` and `dart/lib/main.dart`.
- [ ] The code generator is run and the code is formatted (via `just precommit`).
- [ ] If this PR adds/changes features, documentations (in the `./book` folder) are updated.
- [ ] CI is passing.

## Remark for PR creator

- Justfile is a task runner for the command line interface that allows you to run shell commands from a file named `justfile` in your project directory. You can use Justfile after [installing it](https://github.com/casey/just). Note that commands written in `justfile` of this repository are expected to be run in `bash`, not `cmd` or `powershell`. Running `just ...` commands in `cmd` or `powershell` will produce errors as the syntax is not compatible. On Windows, you can use `git bash` if you have Git installed.
- If fzyzcjy does not reply for a few days, maybe he just did not see it, so please ping him.

### CODE REVIEW

**Comment by fzyzcjy** [comment]
The feature looks good! cc @Desdaemon who implemented the web part

**Comment by LemonHX** [comment]
hello?

**Comment by fzyzcjy** [comment]
@Desdaemon - do you think this PR is OK? If so I will merge it :)

**Review by Desdaemon — CHANGES_REQUESTED** [review]
Just a nit, looks good so far!

File: frb_dart/bin/serve.dart [hunk 1]
```diff
@@ -127,6 +127,11 @@ class Opts {
   late bool runTests;
   @CliOption(help: 'Compile in release mode', negatable: false)
   late bool release;
+  @CliOption(
+      help: 'Use local canvaskit\n'
+          'Firefox does not support remote loaded canvaskit due to CORS settings on the CDN',
+      defaultsTo: true)
```
[the remark this answers is not in the document]
Comment (LemonHX) [reply frb_dart/bin/serve.dart:133]: that will be greater

**Comment by fzyzcjy** [comment]
👀 

(this is mergeable once CI is green)

**Comment by fzyzcjy** [comment]
IMHO this is supported in v2. Thank you all the same for the PR, and feel free to create an issue if it does not work in v2!

**Pull request closed without merging**